scripts/_bootstrap.sh

```diff
@@ -96,6 +97,90 @@ function _bootstrap() {
       echo "Deleted role ${TERRAFORM_ROLE_NAME} and associated policy ${admin_policy_arn}"
     ;;
     #----------------
+    "destroy-non-mgmt")
+      if [[ "$(aws sts get-caller-identity)" != *dev* && "$(aws sts get-caller-identity)" != *NHSDAdminRole* ]]; then
```

Comment:
think this should be an OR?

```suggestion
      if [[ "$(aws sts get-caller-identity)" != *dev* || "$(aws sts get-caller-identity)" != *NHSDAdminRole* ]]; then
```